pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
```

Comment:
this doesn't use setuptools-scm -- the only reason this "works" is because `setuptools-scm` pulls in `packaging` as a transitive dependency

Reply:
good catch